erdpy/validators/validators.py

```diff
@@ -18,18 +26,35 @@ def estimate_system_sc_call(args, base_cost, factor=1):
     return gas_limit
 
 
+def _read_json_file(file_path):
+    val_file = path.expanduser(file_path)
+    guards.is_file(val_file)
+    with open(file_path,  "r") as json_file:
+        try:
+            data = json.load(json_file)
+        except Exception:
+            raise Exception("cannot read validators data")
```

Comment:
We have `errors.py`, with custom errors. Add a new error deriving from `KnownError` perhaps?

Reply:
done.